sklearn/pipeline.py

```diff
@@ -609,6 +609,12 @@ def _transform_one(transformer, X, y, weight, **fit_params):
     return res * weight
 
 
+def _inverse_transform_one(transformer, X, weight, **fit_params):
+    weight = weight or 1
```

Comment:
I'd rather make `1 if weight is None else weight` more explicitly... bool(weight) is not a good way to check for None